### PR TITLE
Enrich Profile texts and translations with more links

### PR DESCRIPTION
**Pull request by arothuis**
This implements #58.

### CODE REVIEW

**Comment by arothuis** [comment]
This is the simplest way of safely adding links to the texts. If we need a more flexible approach, we should consider implementing some sort of HTML whitelist.

**Comment by arothuis** [comment]
@rjkip I added `stiptags` because using `raw` instead would make `replace` pointless as you would just accept HTML at that point.

**Comment by arothuis** [comment]
Removed striptags in favor of a more escape-all approach.

Would you mind reviewing this, @rjkip?

File: app/Resources/translations/messages.en.xliff [hunk 1]
```diff
@@ -1,321 +1,326 @@
...
     <header>
       <tool tool-id="JMSTranslationBundle" tool-name="JMSTranslationBundle" tool-version="1.1.0-DEV"/>
       <note>The source node in most cases contains the sample message as written by the developer. If it looks like a dot-delimitted string such as "form.label.firstname", then the developer has not provided a default message.</note>
     </header>
     <body>
       <trans-unit id="4422839d75319cb038791289059a5c37b6464d3e" resname="profile.application.header">
-        <jms:reference-file line="20">Resources/views/layout.html.twig</jms:reference-file>
         <source>profile.application.header</source>
         <target>Overview of your SURFconext profile</target>
+        <jms:reference-file line="20">/../src/OpenConext/ProfileBundle/Resources/views/layout.html.twig</jms:reference-file>
```
Comment (rjkip) [inline app/Resources/translations/messages.en.xliff]: What the heck is happening here? :sweat_smile:

File: src/OpenConext/ProfileBundle/Resources/views/MyProfile/overview.html.twig [hunk 2]
```diff
@@ -6,8 +6,8 @@
 
 {% block content %}
     <h1>{{ 'profile.my_profile.long_title'|trans }}</h1>
-    <p>{{ 'profile.my_profile.introduction'|trans }}</p>
-    <p>{{ 'profile.my_profile.explanation'|trans }}</p>
+    <p>{{ 'profile.my_profile.introduction'|trans }} <a href="{{ global_view_parameters.attributeInformationUrl }}">{{ 'profile.my_profile.attributes_information_link_title'|trans }}</a>.</p>
+    <p>{{ 'profile.my_profile.explanation'|trans|escape|replace({'%my_services_link%': '<a href="' ~ path('profile.my_services_overview') ~ '">' ~ 'profile.my_services.short_title'|trans ~ '</a>'})|raw }}</p>
```
Comment (rjkip) [inline src/OpenConext/ProfileBundle/Resources/views/MyProfile/overview.html.twig]:
```diff
- ~ 'profile.my_services.short_title'|trans ~
+ ~ 'profile.my_services.short_title'|trans|e ~
```

**Pull request merged**